src/node/blockstorage.cpp

```diff
@@ -364,6 +364,12 @@ bool BlockManager::LoadBlockIndexDB(const Consensus::Params& consensus_params)
         LogPrintf("LoadBlockIndexDB(): Block files have previously been pruned\n");
     }
 
+    // Check whether we have ever pruned block & undo files
```

Comment:
not sure what this comment means.
"ever" could be a misspelt "every". If that's the case, best to change it to from 'every' to 'all the'